docusaurus/docs/reactnative/basics/offline_support.mdx

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Where is the best place to put this tutorial? basics or guides?

Reply:
Spoke with Neil yesterday and we are going with basics. Just need to know if we should put it in any specific order.